shared/AppInsightsCore/src/JavaScriptSDK/AppInsightsCore.ts

```diff
@@ -7,7 +7,6 @@ import {
     ITimerHandler, arrAppend, arrForEach, arrIndexOf, createTimeout, deepExtend, hasDocument, isFunction, isNullOrUndefined, isPlainObject,
     isPromiseLike, objDeepFreeze, objDefine, objForEachKey, objFreeze, objHasOwn, scheduleTimeout, throwError
 } from "@nevware21/ts-utils";
-import { cfgDfMerge } from "../Config/ConfigDefaultHelpers";
```

Comment:
This is an unused import which is now being detected by the re-activated CodeQL checks -- which is blocking other PR's